imports/plugins/core/ui-navbar/client/components/navbar.js

```diff
@@ -27,10 +28,20 @@ class NavBar extends Component {
     shop: PropTypes.object
   }
 
+  constructor(props) {
+    super(props);
+    this.startIntroBtn = this.startIntroBtn.bind(this);
+  }
+
   state = {
     navBarVisible: false
   }
 
+  startIntroBtn(event) {
```

Comment:
You can write this as 
```js
startIntroBtn = (event) => {...}
```
and you won't need the binding in the constructor - no need for constructor